src/Security/Authenticator/ApiKeyAuthenticator.php

```diff
@@ -69,7 +70,7 @@ public function getCredentials(Request $request)
      */
     public function getUser($credentials, UserProviderInterface $userProvider)
     {
-        return new User();
+        return new User(new UuidV6(), 'apiUser', 'notUsed');
```

Comment:
this sounds very strange to me. Can we have a user interface with 2 types of user: ApiUser and InternalUser smt like that?

Reply:
I think it's possible, but it would be too much in this PR I think. I left a TODO comment to address this later. Are you ok with it?